integrations.json

```diff
@@ -868,7 +895,12 @@
           "type": "Datadog.Trace.ClrProfiler.AutoInstrumentation.CosmosDb.DatabaseQueryIteratorsIntegrations",
           "action": "CallTargetModification"
         }
-      },
+      }
+    ]
+  },
+  {
+    "name": "AspNetCore",
```

Comment:
This looks like it was generated incorrectly or there was a bad merge. Can you re-run the generation of integrations.json?

Reply:
Yes, has been regenerated in several commits now.